### DISCUSSION TITLE
How to make TTS sound like a radio transmission

### DISCUSSION THREAD

**Discussion by BitBarrel** [coqui-ai/TTS#855] [General Q&A] [answered]
I want to make TTS sound like it is transmitted over VHF radio (for aviation). What would be the best approach, just run the output through an equalizer effect, or train the model on actual radio transmissions?

**Comment by Sadam1195** [comment, marked as answer]
I would suggest to add effects using any audio tool because in my view actual radio transmission audios are usually distorted, contain a lot of noise and these TTS models are mostly trained and tested on cleaned speech audio dataset. Maybe you could try and train it with actual radio transmissions and see how it works out but I wouldn't recommend.  
@BitBarrel

**Reply by BitBarrel** [reply]
Thanks, makes sense.
